### ISSUE TITLE
Rebranding to another names

### ISSUE THREAD

**Issue by willzyx-hub** [pbatard/rufus#1892]
Hi Rufus Team... So uh... This not an issue but i just wanna to ask can i rebrand rufus application to another names (and ofc i wanna to add some feature also) is that possible by using This source ?

**Comment by pbatard** [comment]
As long as you follow the [GPL v3](https://www.gnu.org/licenses/gpl-3.0.en.html), you can do what you want, including producing your own version of the application under a different name. For instance, that is what the Endless OS people have done for their USB media creation utility (https://github.com/endlessm/rufus/commits/master).

However, be very mindful that you must make any change you apply to the source code 100% public, which means that, you cannot keep the source code for any of the features you add hidden from the public (even temporarily: if you publish a binary with an extra feature, then the code for that extra feature **must** be made public at the same time). You also cannot add features that would make you, the developer, have more freedoms than the users of your software (in other words, you cannot restrict what someone who produces a version of Rufus using your derivative code can and cannot do, compared to your own version).

Basically the GPLv3 means that you really don't even have to ask for permission if you want to create derivative software. But it also implies that anyone can also create derivative software from your source, that includes all the features you added, without asking you.

PS: If you create a derivative, like the Endless OS people did, I'd encourage you to host your project on GitHub and use the fork facility to clone this repository, as it should make things easier for you as well as make your project easier to find (plus it would automatically take care of the GPLv3 requirements with regards to making sure that your source is public).
